### ISSUE TITLE
reviewInputType in mutaion's documantaion

### ISSUE THREAD

**Issue by omaraya11** [graphql-java/graphql-java#488]
I was following the guide to create mutation defining new input type like 

```
GraphQLInputObjectType reviewInputType = GraphQLInputObjectType.newInputObject()
        .name("ReviewInput")
        .field(newInputObjectField()
                .name("stars")
                .type(Scalars.GraphQLString)
                .name("commentary")
                .type(Scalars.GraphQLString))
        .build();
```

but it didn't work till i changed it like this

```
GraphQLInputObjectType reviewInputType = GraphQLInputObjectType.newInputObject()
        .name("ReviewInput")
        .field(newInputObjectField()
                .name("stars")
                .type(Scalars.GraphQLString))
        .field(newInputObjectField()
                .name("commentary")
                .type(Scalars.GraphQLString))
        .build();    
```

I'm not sure if the documentation is wrong so it would be good to corrected or i did something wrong following the guide.

**Comment by bbakerman** [comment]
yup that seems to be missing the extra field statement

(which is why I like IDL for defining Schema as an aside)

**Comment by andimarek** [comment]
@omaraya11 could you create a PR for that? That would be awesome ... thanks!

**Comment by omaraya11** [comment]
Sure .. anytime

**Comment by andimarek** [comment]
closes with #513
